packages/plugin-core/src/workspace/tutorialInitializer.ts

```diff
@@ -54,14 +61,22 @@ export class TutorialInitializer
 
     fs.copySync(path.join(dendronWSTemplate.fsPath, "tutorial"), vpath);
 
+    const ABUserGroup = MEETING_NOTE_TUTORIAL_TEST.getUserGroup(
+      SegmentClient.instance().anonymousId
+    );
+
+    const replaceString =
+      ABUserGroup === MeetingNoteTestGroups.show ? MeetingNoteTutorialText : "";
+
     // Tailor the tutorial text to the particular OS and for their workspace location.
     const options = {
       files: [path.join(vpath, "*.md")],
 
-      from: [/%KEYBINDING%/g, /%WORKSPACE_ROOT%/g],
+      from: [/%KEYBINDING%/g, /%WORKSPACE_ROOT%/g, /%MEETING_NOTE_CONTENT%/g],
```

Comment:
this works but i think moving forward, we should think of just having different zip files with different tutorials.
instead of modifying tutorial in text, just swap out with completely different tutorial (its okay if 90% of text is the same)

the reason is we still need to sync the updated tutorial with the workspace tutorial. the 80/20 way of doing this is to run `export pod` and dump the contents.
showing native keybindings depending on the operating system is nice but makes this process more complex because we can't just use the exported docs. 

to simplify syncing docs and ab test, i would propose just doing a/b tests with completely different doc sets